### ISSUE TITLE
locate_motion_mode on does not show red box in overlay when storage a video file

### ISSUE THREAD

**Issue by Metus88** [Motion-Project/motion#1648]
### Did you read the guide?

Yes

### What is the base version number of Motion being used?

4.2.x

### What was the install method?

Installed via package tool

### What is base architecture?

Other

### What is the distro being used?

Ubuntu

### Disto version number

DOCKER IMAGE: OS Version Ubuntu 18.10 motionEye Version 0.42 Motion Versios 4.2.2	

### Camera type(s) being used?

Network camera with RTSP

### Describe the issue/problem and steps to reproduce

I used the docker image "ccrisan/motioneye:master-amd64" of motioneye with this config file of docker compose:
```
services:
  motioneye:
    container_name: motioneye
    image: ccrisan/motioneye:master-amd64
    volumes:
    - "/etc/localtime:/etc/localtime:ro"
    - "/home/deb/DOCKER/motioneye:/etc/motioneye"
    - "/home/deb/DOCKER/motioneye/storage:/var/lib/motioneye"
    network_mode: host
    restart: unless-stopped
```
The operation system of the mini pc where docker is installed it is a Debian 11.6.0  amd64

If you configure a IP Network camera  RTSP and under "Video Device"->"Extra Motion Options" you write "locate_motion_mode on" and save, the video file stored when a motion is detected doesn't show the red lines/box overlay that I suppose have to appear. (I tried also with and without enable Text Overlay option.)
The documentation has to be update [locate motion mode](https://motion-project.github.io/motion_config.html#locate_motion_mode) or it's a bug?

### Relevant Motion log output (at log_level 8)

```shell
Before send my log level 8 I would like to know if it's a bug or only my misunderstanding of documentation.
Thanks
```

**Comment by zagrim** [comment]
@Metus88 your link for "locate motion mode" seems odd, wrong link perhaps?

**Comment by Mr-Dave** [comment]
It looks like it is working for me.  (Saving the red box around the changes in the saved videos).  Did you specify both mode and style?  e.g.
```
locate_motion_mode on
locate_motion_style redbox
```

**Comment by zagrim** [comment]
I have the same issue, same Motion version, and I have both of those in my config. The picture saved for motion event has the red box, but the video does not have any. And I, too, have RTSP cameras.

One thing that might actually make a difference in my case, though, that I didn't think of before: I'm using `movie_passthrough on` (as well as `netcam_highres`), would either or both of those cause the video to be without redboxing?

**Comment by Mr-Dave** [comment]
That is correct.  The purpose of `movie_passthrough` is to avoid that extra processing for overlays, etc.  

It is mentioned in the  documentation for [movie_passthrough](https://motion-project.github.io/motion_config.html#movie_passthrough)

> No image processing is performed so text overlays, privacy masks etc will not be on the resulting video.

**Comment by zagrim** [comment]
@Metus88 Do you by chance use `movie_passthrough`? In ME 0.42 (MEOS) I think you'd have put that in Extra Motion Options since that release didn't yet have that feature directly supported in the UI...

**Comment by Metus88** [comment]
> @Metus88 your link for "locate motion mode" seems odd, wrong link perhaps?

I edited it, thanks my mistake. Wrong link sorry

**Comment by Metus88** [comment]
> @Metus88 Do you by chance use `movie_passthrough`? In ME 0.42 (MEOS) I think you'd have put that in Extra Motion Options since that release didn't yet have that feature directly supported in the UI...

Yes, I used "movie_passthrough on". Now if I set it off, the redbox appears!
Sorry I did not know about it. Now it's clear. However I will reactivate it (movie_passthrough on) because with "movie_passthrough off" I have a crop of the video (like zoom in) and the clips from 1MB size they become to 20MB size (same duration). Probably I have to adjust other parameters.
I will close the issues. Thanks @Mr-Dave and @zagrim for the help!
